src/types/wrappers/EqualGrowthCommunityModel.jl

```diff
@@ -0,0 +1,97 @@
+"""
+$(TYPEDEF)
+
+A wrapper around [`CommunityModel`](@ref) that returns a community model where
+the growth rates of all members are constrained to be equal to
+`community_objective_id`, which is the community growth rate. The objective of
+the resultant model is set to this `community_objective_id`.
+
+# Notes
+1. No biomass metabolite exists (and none are created).
+
+# Fields
+$(TYPEDFIELDS)
+"""
+Base.@kwdef mutable struct EqualGrowthCommunityModel <: AbstractModelWrapper
+    inner::CommunityModel
+    community_objective_id::String = "community_biomass"
```

Comment:
is it objective id or reaction id?

Reply:
this is a special kind of variable, not really a reaction in this context